docs/EmptyState.md

```diff
@@ -53,10 +53,12 @@ The following child elements are projected into `<pxb-empty-state>`:
 
```

Comment:
^^ Title is no longer a required @Input ^^

Reply:
Good catch, I've updated the docs and will merge this PR in.